go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/googleapis/gnostic v0.3.1 // indirect
 	github.com/imdario/mergo v0.3.8
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
+	github.com/kubernetes-sigs/kube-storage-version-migrator v0.0.0-20191127225502-51849bc15f17
```

Comment:
I think that we should update https://github.com/openshift/cluster-kube-apiserver-operator/blob/4d26538c218b348a2e3376bfdb402bd8350d5485/go.mod#L36 instead

Reply:
unless we don't need our fork then we should remove an entry from `replace` directive.

Reply:
upstream seems to be using `1.17.0` - https://github.com/kubernetes-sigs/kube-storage-version-migrator/blob/master/Gopkg.toml

Reply:
Yes, you are right! we can remove our fork if it is not needed for this release.

Reply:
Upstream does not have the hardcoded qps/burst values, does it?

Reply:
We import it here for the client. As I understand it, the qps/burst settings were hard coded for the actual migrator. So.. technically, yes we don't need the replace as we are only using the client here. The migrator image though, needs to be from our fork.